src/fpm.f90

```diff
@@ -65,7 +65,7 @@ subroutine build_model(model, settings, package, error)
     end if
 
     call new_compiler(model%compiler, settings%compiler, settings%c_compiler, &
-        & echo=settings%verbose, verbose=settings%verbose)
+        & settings%cpp_compiler, echo=settings%verbose, verbose=settings%verbose)
```

Comment:
This should be consistent with `cxxflags`, so either `cxx_compiler` or `cppflags`.